frontend/includes/detail.js

```diff
@@ -57,7 +57,7 @@ class HotelNameLocation extends React.Component{
     const { image, addresses, coordinates, name} = this.state;
 
     const hotelImage = { backgroundImage: `url(${image})`, };
-    const mapLink = coordinates ? `https://www.google.com/maps?q=${coordinates[1]},${coordinates[0]}` : "javascript:void(0)";
+    const mapLink = coordinates ? `https://www.google.com/maps?q=${coordinates[0]},${coordinates[1]}` : "javascript:void(0)";
```

Comment:
Is new `coordinates` value a pair of `(lon, lat)`?

Reply:
Before it was was (lon, lat) now it is (lat, lon).

Reply:
@cristianc-ty , are you sure this was correct? The map URL in https://trustyou.github.io/ota-demo/frontend/details.html?ty_id=648f8465-620a-4c33-83fc-e891a3da36dd puts me somewhere in the Arab sea, but it is placed correctly in the search results

Reply:
Fixed.